winui/AutoComplete/No-Result-Found.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: No Result Found in WinUI AutoComplete control | Syncfusion
+description: Learn how to use NoResultsFoundTemplate and NoResultsFoundContent in Syncfusion WinUI AutoComplete control.
+platform: winui
+control: SfAutoComplete
+documentation: ug
+---
+
+# No Result Found  in WinUI AutoComplete (SfAutoComplete)
+
+When the entered item is not in the suggestion list, SfAutoComplete displays a text indicating there is no search results found. We can set the desire text to be displayed for indicating no results found with the [NoResultsFoundContent] property and  the desire text to be displayed for indicating no results found can be customized using the [NoResultsFoundTemplate] property. 
+
+## NoResultsFoundContent 
+
+The below code shows how to include the `NoResultsFoundContent` in AutoComplete.

```

Comment:
@YuvarajMasilamani, include more details in this topic about the use case of the properties, example, screenshot. Compare competitor and our other platform.

Reply:
resolved